cli/src/account.rs

```diff
@@ -52,11 +53,16 @@ pub async fn account_prompt_internal(account: Account, history: &mut AccountHist
         .completion_with(&ACCOUNT_COMPLETION)
         .interact_text()?;
     match command.as_str() {
-        "h" => print_account_help(),
-        "clear" => {
+        "h" | "help" => print_account_help(),
+        "c" | "clear" => {
```

Comment:
Probably I miss something but don't know yet :D.  Wouldn't be easier to extend `WalletCommand`? https://github.com/iotaledger/iota-sdk/blob/8c0819f40cdf4fb1b41c8be2070a2b7d8b32cea3/cli/src/wallet.rs#L24

Reply:
This is about the "list" match arm that I added, right? I can totally imagine it being a `WalletCommand` as well or even instead. Do you think it's enough to get the account aliases at the command-line (non-interactive), or is it better to get them when you're already logged into an account (interactive) like it is now?

Reply:
I can imagine a use case when someone forgot the names of accounts. So they can use `wallet list-accounts`, find the name and the `wallet main` So probably, non-interactive mode would be better?